README.md

```diff
@@ -292,14 +308,15 @@ Provide multiple paths/patterns:
 
 ```js
 {
-  title: 'Multiple Files',  
+  title: 'Multiple Files',
   files: [
     'my/project/styles/global/*',
     'my/project/styles/components/feedback.scss',
     '!my/project/styles/global/colors.scss'
   ]
 }
 ```
+*`!` negates the match*
```

Comment:
We are using globby for all files declarations. Pattern matching, in general, should be linked to globby. Do you think we should include explanations like line 319? Then we should explain line 313 (`*`). Should we give pattern matching its own section in addition to a globby link?